homework-4/matrix.py

```diff
@@ -0,0 +1,13 @@
+matrix=[[1,4,7,12,15,1000],
+    [2,5,19,31,32,1001],
+    [3,8,24,33,35,1002],
+    [40,41,42,44,45,1003],
+    [99,100,103,106,128,1004]
+]
+
+for i,j in enumerate(matrix):

```

Comment:
This is a good attempt at solving the problem! The only issue is if we want the target to be something other than 44 which is hardcoded at the moment. The problem is asking for a function that accepts a matrix of numbers and a target value then returns the location of the target value in the matrix. Just need to put this solution into a function :) Additionally, you can go one step further and refactor this solution to use a list comprehension to shorten and improve its performance. All in all good work.